doc/getting-started.md

```diff
@@ -49,14 +49,23 @@ make setup-external-dns-test
 make deploy-cert-manager
 ```
 
+Alternatively the following make target wraps all of above commands:
+```
+make setup-getting-started-cluster
+```
+
 ## Deploy gateway-controller
 
 There are three alternative ways to deploy the controller as described
 by the following sections.
 
 ### Deploy with Helm (recommended)
 
-TODO: We do not yet have a Helm chart
+Deploying the controller using Helm can be sone as follows
+```
+helm repo add tv2-oss https://tv2-oss.github.io/gateway-controller
+helm upgrade -i gateway-controller tv2-oss/gateway-controller --create-namespace
```

Comment:
The controller defaults to namespace `gateway-controller-system`, i.e. this is where it looks for 'global' policies and this is given through command line argument `controller-namespace`. I would gues the getting-started guide works because there are no global policies used, but it could be an good idea to use this 'standard' namespace name - i.e. the same as the make target above.